bin/iotAgentTester.js

```diff
@@ -17,7 +17,7 @@
  *
  * You should have received a copy of the GNU Affero General Public
  * License along with fiware-iotagent-lib.
- * If not, seehttp://www.gnu.org/licenses/.
+ * If not, see http://www.gnu.org/licenses/.
```

Comment:
This change was already included in a PR merged yesterday. Maybe something has gone wrong with some merge...

Reply:
Outdated

Reply:
NTC